{{cookiecutter.__project_slug}}/README.md

```diff
@@ -0,0 +1,169 @@
+# {{ cookiecutter.project_name }}
+
+**Owner:** {{ cookiecutter.author_name }} ({{ cookiecutter.author_email }})
+
+## Scope
+
+In this section we describe why this sofware was built and give a quick overview of the system.
+
+### Purpose
+<!--
+Describe the purpose of your software project. Describe why and in which context you are writing this software. Describe what your software will and will not do.
+-->
+{{ cookiecutter.project_purpose }}
+
+### System Overview
+<!--
+Describe the total system this piece of software is a part of and where in this system this fits in. For a lot of simple scripts this does not have to be very complicated.
+-->
+{{ cookiecutter.system_overview }}
+
+## User Documentation
+<!--
+This user documentation contains information on how to get started with the script that is contained in the basic templates.
+Extend this section with specific instructions for users of your software (e.g. what format should the input data be in? What kind of settings can be changed? ...)
+-->
+
+### Pre-requisites
+
+To use this script you first need to install [python](https://www.python.org/) (version 3.10 or higher) and [poetry](https://python-poetry.org/). The installation guide also assumes you have [git](https://git-scm.com/) installed.
+
+### Installation
+
+To install this software first make a local copy of this repostitory (e.g by cloning it with `git clone`). In the root directory of your local folder run the following commands:
```

Comment:
Maybe make the second sentence a separate paragraph. This poetry install command must be done after cloning the repo succesfully.